app/services/calculation_orchestrator.rb

```diff
@@ -1,25 +1,20 @@
 # frozen_string_literal: true
 
 require "participant_event_aggregator"
-require "contract_event_payment_calculator"
+require "payment_calculator/ecf/payment_calculation"
 
 class CalculationOrchestrator
   class << self
-    def call(lead_provider:, aggregator: ::ParticipantEventAggregator, calculator: ::ContractEventPaymentCalculator, event_type: :started)
-      new(lead_provider: lead_provider, aggregator: aggregator, calculator: calculator).call(event_type: event_type)
+    def call(lead_provider:,
+             contract:,
+             aggregator: ::ParticipantEventAggregator,
+             # TODO: Add in aggregator for Uplift as uplift_aggregator: ::ParticipantUpliftAggregator (for example)
```

Comment:
I suggest adding the jira ticket link to the TODO if this is referring to https://dfedigital.atlassian.net/browse/CPDTP-195
That way it's clear how this todo will be resolved.